spec/lower_level_api/integration/basic_consume_spec.rb

```diff
@@ -96,4 +96,10 @@
       ch.close
     end
   end
+
+  it "propagates the connection_name to the client_properties" do
```

Comment:
This belongs to connection tests. It has nothing to do with consumers.

Reply:
done